.github/workflows/prod-app-1-cd.yml

```
@@ -0,0 +1,117 @@
+name: Prod App 1 CD
+on:
+  workflow_call:
+    inputs:
+      run_id:
+        required: true
+        type: string
+    secrets: inherit
```

actionlint (1.7.7)

"secrets" section is scalar node but mapping node is expected

(syntax-check)